baseframe/forms/validators.py

```diff
@@ -319,6 +320,15 @@ def __call__(self, form, field):
             return
         else:
             raise ValidationError(self.message)
+            
+class IsEmoji(object):
```

Comment:
This validator was inserted between two validators that are related to each other. This one should have been before or after them. Move it.